docs/upgrade/focal_migration.rst

```diff
@@ -0,0 +1,309 @@
+  ssh mon hostname     # command output should be 'mon'
+
+If you are having trouble accessing the servers via SSH, try the following:
+
+- create a new Tor network circuit by disconnecting and reconnecting your
+  Internet link, and repeat the check
+- run the ``./securedrop-admin tailsconfig`` command and repeat the check
+- verify that the *Source* and *Journalist Interfaces* are available via their
+  desktop shortcuts
+- verify that the *Application* and *Monitor Servers* are up
+- :ref:`contact us <contact_us>` for assistance.
+
+.. _choose_migration_path:
+
+Choose Migration Path
+---------------------
+
+If your instance is already using v3 onion services, choose our
+:ref:`Standard Migration Procedure <migration_standard>`.
+
+Instances that have not yet enabled v3 
+:ref:`onion services <glossary_onion_service>` should choose the 
+:ref:`Alternate Migration Procedure <migration_alternate>`. 
+
+
+.. _prune_submissions:
+
+Delete Old Submissions from the Server
+--------------------------------------
+
+In coordination with journalists, ensure that any old or unneeded
+submissions have been deleted from the server. Pruning old submissions
+will reduce the size and improve the speed of your server backup. 
+Journalists can delete unneeded submissions via the *Journalist Interface.*
+
+
+.. _download_focal:
+
+Download and Verify Ubuntu 20.04 LTS (Focal) Installation Media
+----------------------------------------------------------------
+
+Follow our instructions to 
+:doc:`download and verify Ubuntu Server 20.04 LTS <../servers>` and
+install the .iso file onto a USB stick.
+
+You have now completed all the preparatory steps. The rest of the 
+migration procedure will be completed during your maintenance window. 
+
+Migration
+~~~~~~~~~ 
+
+.. _migration_standard:
+
+Standard Migration Procedure
+----------------------------
+Perform these steps if your instance is already using v3 onion services. Ensure
+you have completed the :ref:`preparatory steps <focal_prep>`.
+
+#. Announce your maintenance window. As part of this procedure, your servers
+   will become unreachable.
+#. :doc:`Take a backup of the current instance <../backup_and_restore>`. Once 
+   you have taken a backup of the servers, power them off.
+
+    .. warning::
+
+     The next steps will overwrite existing data on the servers.
+
+     Data from the *Monitor Server* will not be restored after the backup.
+     If you require historical data from the *Monitor Server*, archive it
+     separately before proceeding.
+#. Follow the instructions on 
+   :ref:`hardware migration for instances using v2+v3 or v3 onion services <migrate_v3>`. 
+
+   You will be instructed to install Ubuntu 20.04, overwriting the 
+   existing operating system on the servers,
+   and you will be guided through reinstalling SecureDrop.
+
+   Once you have reinstalled SecureDrop, you will be instructed on how to
+   apply your backup file, restoring data and Tor credentials.  
```

Comment:
I would simplify this step, e.g.:

> Follow the instructions for migrating an instance with v3 onion services. As part of this process, you will reinstall your servers, restore your backup, and configure access via your *Admin Workstation*.

Reply:
Sure, tweaked in  b9367dc. I wanted to provide a bit of detail just to give people confidence that yes, wiping and reinstalling from scratch is what we want them to do. I think this accomplishes that goal.

Reply:
It does! I would like to avoid the mouthful `v2+v3 or v3 onion services` wherever possible, but that's a minor nit and history will solve this problem for us soon. :)